tests/unit_tests/test_qmpc.py

```diff
@@ -14,14 +14,47 @@ class TestQMPC:
     )
 
     @pytest.mark.parametrize(
-        ("secrets"), [([[1, 2, 3]]), ([[1], [2], [3]]), ]
+        ("secrets", "schema"), [
+            ([[1, 2, 3]], ["attr1", "attr2", "attr3"]),
+            ([[1], [2], [3]], ["attr1"]), ]
     )
-    def test_send_shares(self, secrets, run_server1, run_server2, run_server3):
+    def test_send_shares(self, secrets, schema,
+                         run_server1, run_server2, run_server3):
         """ serverにシェアを送れるかのTest"""
         response: Dict[str, Any] = self.qmpc.send_share(
-            secrets, [], 1, 1000)
+            secrets, schema, 1, 1000)
         assert (response["is_ok"])
 
+    def test_send_shares_errorhandring(self,
+                                       run_server1, run_server2, run_server3):
+        with pytest.raises(Exception):
```

Comment:
I would like `Exception` to be more accurate.
For example, `RuntimeError` and `ArgmentError`....